content/quickstart.md

```diff
@@ -345,3 +345,97 @@ and sends invoices to clients on the same schedule.
 
 [harvest]: https://www.getharvest.com/
 [accounts]: mailto:accounts@oddbird.net
+
+## Success Criteria:
+
+This guide will help you understand the expectations for your role at OddBird.
+The criteria are divided into three groups: Work Quality, Communication, and
+Learning and Growth. Each group has three levels of performance: Under performs,
+Meets, and Exceeds. You can use this guide to assess your own performance,
+identify your strengths and areas for improvement, and set your goals and action
+plans.
+
+### Work Quality
+
+#### Under performs
+
+- Lacks initiative and waits for directions or assignments on high priority work
+- Produces low-quality or incomplete work that requires excessive rework or
+  refactoring from others
+- Uses outdated or inappropriate tools, technologies, or methodologies
+- Avoids or gives up on challenges
+- Does not plan or dissect big assignments into smaller tasks
+
+#### Meets
+
+- Takes initiative on high priority work by chipping away at it
+- Delivers complete and high-quality work and acts on feedback from others
+- Approaches challenges by spending time planning and dissecting out assignments
+
+#### Exceeds
+
+- Takes initiative planning and breaking down upcoming work
```

Comment:
```suggestion
- Takes the initiative and leads others in planning and breaking down upcoming work
```
I recommend moving this bullet point to the bottom of this list since the analogous point in the other 2 sections is the last one in those lists. Also this one seems quite similar to the analogous bullet point under "Meets," so I added a bit more.